packages/theme/src/components/editable.ts

```diff
@@ -9,6 +9,7 @@ const baseStylePreview: SystemStyleObject = {
   py: "3px",
   transitionProperty: "common",
   transitionDuration: "normal",
+  minWidth: "2px",
```

Comment:
What's the reason behind using "2px"? It seems arbitrary

Reply:
"2px" was selected so that the pointer has a small width to click on. 

On second thought, 
since the width of `EditableInput` is set to `"full"`, we could also set the width of `EditablePreview` the same?
`minWidth` can be removed in this case.